default.nix

```diff
@@ -27,5 +27,5 @@ in {
     inherit rev;
   };
 
-  module = import ./module.nix flake-self;
+  module = lib.modules.importApply ./module.nix flake-self;
```

Comment:
TIL about [`importApply`](https://noogle.dev/f/lib/modules/importApply), I'll definitely be using this in more of my Nix code from here on forth